bin/web-push.js

```diff
@@ -0,0 +1,45 @@
+#! /usr/bin/env node
+const webPush = require('web-push');
+webPush.setGCMAPIKey(process.env.GCM_API_KEY);
+
+const argv = require('minimist')(process.argv.slice(2));
+
+const usage = 'Use: web-push --endpoint=<url> --key=<browser key> [--auth=<auth secret>] [--ttl=<seconds>] [--payload=<message>] [--vapid]';
+
+if (!argv['endpoint'] || !argv['key']) {
+  console.log(usage);
+  process.exit(1);
+}
+
+const endpoint = argv['endpoint'];
+const key = argv['key'];
+const ttl = argv['ttl'] || 0;
+const payload = argv['payload'] || '';
+const auth = argv['auth'] || null;
+const useVAPID = argv['vapid'] || false;
+
+var params = {
+  TTL: ttl,
+  payload,
+  userPublicKey: key
+};
+if (useVAPID) {
```

Comment:
We should make the VAPID info configurable, so the user can define their own `audience` and `subject` and provide their keys.
We can do it in a follow-up.